vulnfeeds/cves/versions.go

```diff
@@ -52,35 +53,115 @@ type CPE struct {
 	Other      string
 }
 
-func extractGitHubCommit(link string) *FixCommit {
-	// Example: https://github.com/google/osv/commit/cd4e934d0527e5010e373e7fed54ef5daefba2f5
-	u, err := url.Parse(link)
+// Returns the base repository URL for supported repository hosts.
+func repo(u string) (string, bool) {
```

Comment:
Can we make this public and merge the code you have in https://github.com/google/osv.dev/blob/4724b9229883b104ffea7fd71e49b16e3a33819b/vulnfeeds/cpp/main.go#L42, and just call this there?

Reply:
Yes, that was my intention once this has landed. I'll make them public from the get go.